src/frontend/src/mock/teams.ts

```diff
@@ -0,0 +1,8 @@
+import { Team } from '../api/team-api';
+
+export const teams: Team[] = [
```

Comment:
gjerne legg til et ekstra team så man faktisk får testet å bytte team også lokalt. Peder har også solgt meg litt på ideen av å bruke genererte mock-data i stede for manuelt. Da er det lettere å legge til flere / færre ved behov.

For å kunne lage random ting som er forutsigbart har jeg laget egne random-utils med støtte for seeding (for JS har tydeligvis ikke random-seeds...).